imgparse/imgparse.py

```diff
@@ -708,11 +708,39 @@ def get_ils(image_path, exif_data=None, xmp_data=None):
     try:
         make, model = get_make_and_model(image_path, exif_data)
         xmp_tags = xmp.get_tags(make)
-        return parse_seq(xmp_data[xmp_tags.ILS], float)
+        if make == "DJI":
+            return [float(xmp_data[xmp_tags.ILS])]
```

Comment:
The DJI Sun sensor tag is not written the same, it does not need the parse sequence function.  Not sure if this is the correct way to handle this, maybe the parse sequence function should handle situations where tags are written differently.

![image](https://github.com/user-attachments/assets/7ec56515-2453-4e4e-b391-f62ff2d3f974)
![image](https://github.com/user-attachments/assets/85cb091b-11c7-43a1-a521-902355120ebc)

Reply:
Yep this looks good